kafka-connect-s3/src/main/java/io/confluent/connect/s3/continuum/NewFileCommittedMessageBody.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Confluent Inc.
```

Comment:
I had included these copyright headers because of a `checkstyle` rule which was checking for a Confluent copyright header in each file. I disabled the check for our files, and replaced "Confluent Inc." with "National Instruments Inc."